scality_manila_utils/cli.py

```diff
@@ -182,10 +182,20 @@ def main(args=None):
         help='Filesystem to get information about'
     )
 
+    parser_wipe = subparsers.add_parser(
+        'wipe',
+        help='Remove an existing export. The export must not have any grants.'
+    )
+    parser_wipe.add_argument(
+        'export_name',
+        help='Filesystem to remove'
```

Comment:
Yes, I don't mind. I think we should be consequent though. Filesystem is used throughout the cli help. If we use share here, we should use it everywhere.